platform/linuxbsd/display_server_x11.cpp

```diff
@@ -2157,6 +2152,9 @@ void DisplayServerX11::_window_changed(XEvent *event) {
 	}
 
 	WindowData &wd = windows[window_id];
+	if (window_id == MAIN_WINDOW_ID)
```

Comment:
What is this for? not quite sure why the main window should not notify size changed

Reply:
Because when you close a window and remove it from the windows map, all _window_changed events were going to the MAIN_WINDOW and making a lot of weird things.